diag_manager/fms_diag_file_object.F90

```diff
@@ -67,6 +67,7 @@ module fms_diag_file_object_mod
   TYPE(time_type) :: next_output      !< Time of the next write
   TYPE(time_type) :: next_next_output !< Time of the next next write
   TYPE(time_type) :: no_more_data     !< Time to stop receiving data for this file
+  logical         :: done_writing_data!< .True. if finished writing data
```

Comment:
This is for the diag file object.
I think there are two cases where this will be set to true:
1. Static files after the first write
2. Files that are using the `file_duration` functionality in the yaml file

Reply:
Can you add that to the documentation